CHANGELOG.rst

```diff
@@ -1,6 +1,11 @@
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Changelog for package swri_console
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+1.0.1 (2017-04-27)
```

Comment:
This line should be `Forthcoming` until a new version is actually released to the ROS build farm.